_layouts/post.html

```diff
@@ -37,8 +49,20 @@ <h1 class="title page-title">{{page.title}}</h1>
     <div class="generated-content">
         {{content}}
     </div>
+
+<hr class="divider" />
 </article>
 
+
+<div class = "social-buttons" style="text-align: center">
+Enjoyed the read? Share on social media! <br>
+<a class="fb-share-button"  data-layout="button" data-size="large" target="_blank" href="" class="fb-xfbml-parse-ignore">Share</a>
+<a class="twitter-share-button" href="https://twitter.com/share?ref_src=twsrc%5Etfw" data-size="large" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Siimlar comment on the script tag as above; also can we `defer` it? Not sure if that causes problems!